contracts/src/ethereum/TokenBridge.sol

```diff
@@ -59,9 +59,13 @@ contract TokenBridge is
 
     function processSubContractAddresses(bytes calldata subContractAddresses) internal override {}
 
+    function isValidFelt(uint256 felt) internal returns (bool) {
+        return (felt != 0) && (felt < CairoConstants.FIELD_PRIME);
```

Comment:
I think this is a bit confusing. 0 is a perfectly valid felt

Reply:
indeed, will change

Reply:
Could also rename it to is validL2Address